src/components/InventoryTable/Pagination.test.js

```diff
@@ -56,7 +56,7 @@ describe('Pagination', () => {
             const store = mockStore(initialState);
             const wrapper = render(<WrappedPagination store={ store } />);
             expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('button[disabled=""]').length).toBe(5);
+            expect(wrapper.find('button[disabled=""]').length).toBe(4);
```

Comment:
I dug around Patternfly changes, I couldn't find anything definitive, but something seems to have changed with this PR https://github.com/patternfly/patternfly-react/pull/7192

The pagination still looks and works the same:

![Screenshot 2022-07-22 at 12 33 17](https://user-images.githubusercontent.com/7757/180422211-c53c3405-f5ef-4626-8d9c-ed707af789eb.png)